### DISCUSSION TITLE
Paths starts with capital letter on heroku

### DISCUSSION THREAD

**Discussion by IonutRealDevs** [nuxt/nuxt#9605] [Questions (Nuxt 2)]
Hello.
I'm facing a problem when i deploy my app on heroku. On localhost everything works good, but on heroku, some paths starts with first letter capital like this `/Account`.
I have this button
`<v-btn icon :to="localePath('/account')"><v-icon>mdi-account</v-icon></v-btn>`
On localhost it sends me to this route `/account` but on heroku it sends me on `/Account` and page never load.
What cause this problem? Thank you

**Comment by kissu** [comment]
It does the same when you `yarn build` or `yarn generate` and `yarn start` locally?
Also, what if you try without i18n (`localePath`)?

**Reply by IonutRealDevs** [reply]
I tried `npm run build` then `npm start` locally and it worked.
Looks like without `localePath` it works. I think this function capitalize the paths. Is there a way you fix that using `localePath`?

**Reply by kissu** [reply]
The function is probably not capitalizing the path but looking for your i18n version of it. If you don't have any, I guess it fallbacks to `Account` indeed.

Give a read to [this documentation](https://i18n.nuxtjs.org/routing) and also, check if you do have a working `account` variant, like `Compte` (in 🇫🇷) in your other locale(s).
